<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMidiNote</class>
 <widget class="QDialog" name="AddMidiNote">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Midi Note</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <widget class="QListWidget" name="listWidgetNotes"/>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButtonAdd">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Add manually</string>
     </property>
     <property name="icon">
      <iconset resource="EProDrums.qrc">
       <normaloff>:/Add.png</normaloff>:/Add.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButtonDel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Del</string>
     </property>
     <property name="icon">
      <iconset resource="EProDrums.qrc">
       <normaloff>:/Remove.png</normaloff>:/Remove.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButtonPrev">
       <property name="text">
        <string>&lt;&lt; Prev</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonNext">
       <property name="text">
        <string>Next &gt;&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="labelHelp">
     <property name="text">
      <string>Please hit a pad or add a note manually</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="EProDrums.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddMidiNote</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddMidiNote</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
